### PR TITLE
qpbo recipe

### PR DESCRIPTION
**Pull request by DerThorsten**

### CODE REVIEW

File: recipes/qpbo/recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/qpbo  # [unix]
+    - test -f ${PREFIX}/include/qpbo/qpbo.hpp  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\qpbo\qpbo.hpp (exit 0) else (exit 1)  # [win]
+
+about:
+  home: https://github.com/DerThorsten/qpbo
+  license: GPL
```
Comment (jakirkham) [inline recipes/qpbo/recipe/meta.yaml]: Please add the version here. Looks like `GPL3`.

File: recipes/qpbo/recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  license_family: GPL
+  license_file: LICENCE
+  summary: 'a header only version of Qpbo/ Qpbo solver for discrete graphical models'
+  description: 'a header only version of Qpbo/ Qpbo solver for discrete graphical models'
```
Comment (jakirkham) [inline recipes/qpbo/recipe/meta.yaml]: If the `description` is redundant, feel free to drop it. Only the `summary` is required. The `description` is a nice to have if it helps explains things further.

File: recipes/qpbo/recipe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/qpbo  # [unix]
```
Comment (jakirkham) [inline recipes/qpbo/recipe/meta.yaml]: I guess we know this is true if we found the header below, right? No harm in having it, but it would be ok to drop.

**Comment by jakirkham** [comment]
Couple minor comments. Otherwise LGTM. Any other thoughts from @conda-forge/staged-recipes?

**Comment by dougalsutherland** [comment]
So is this your fork of the software? I guess the "upstream" project is just a tar file on a professor's site, but I'm always a little bit leery of publishing small forks under the original name....

**Comment by DerThorsten** [comment]
@dougalsutherland  I can rename my github repo to some different name if this would be  appreciated

**Pull request closed without merging**